libs/openFrameworksCompiled/project/makefileCommon/config.linux.common.mk

```diff
@@ -196,7 +196,7 @@ endif
 
 ifndef PROJECT_OPTIMIZATION_CFLAGS_DEBUG
 	# DEBUG Debugging options (http://gcc.gnu.org/onlinedocs/gcc/Debugging-Options.html)
-	PLATFORM_OPTIMIZATION_CFLAGS_DEBUG = -g3
+	PLATFORM_OPTIMIZATION_CFLAGS_DEBUG = -g0
```

Comment:
hm, I guess this change slipped in during commit, as that is exactly what you do with the `sed` below?